docs/commcare-cloud/backup.md

```diff
@@ -229,10 +229,25 @@ The `blobdb` is our binary data store.
 BlobDB backups create a compressed version of the blobdb data directory.
 
 ### Restoring BlobDB Backups
+The BlobDB restore process is the same as the couchdb restore process in that it involves extracting the backed up data
```

Comment:
Maybe add a note that this is true for the file system blob db (is that the default?), but that if you are using some other distributed S3 compatible blob system (e.g. Minio), that you should follow the instructions provided by that system.

Reply:
@proteusvacuum Ah, good point.